app/containers/BestNewMusicCardContainer/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { removeQuotes } from '../../utils';
+import * as FavoritesActions from '../../actions/favorites';
+import * as ToastActions from '../../actions/toasts';
+import { getFavoriteTrack } from '../../selectors/favorites';
+import BestNewMusicCard, { bestNewItemShape } from '../../components/Dashboard/BestNewMusicTab/BestNewMusicMenu/BestNewMusicCard';
+
+const BestNewMusicCardContainer = props => <BestNewMusicCard {...props} />;
+
+BestNewMusicCardContainer.propTypes = {
+  item: bestNewItemShape,
+  onClick: PropTypes.func
+};
+
+BestNewMusicCardContainer.defaultProps = {
+  item: null,
+  onClick: () => { }
+};
+
+function mapStateToProps(state, ownProps) {
+  return {
+    favoriteTrack: ownProps.item
+      ? getFavoriteTrack(state, ownProps.item.artist, removeQuotes(ownProps.item.title))
```

Comment:
Here I'm removing the quotes that are coming from `pitchfork`. We can consider removing the directly after fetching data to avoid any future problems.